src/lib/Scenes/Artwork/__tests__/Artwork-tests.tsx

```diff
@@ -111,8 +111,10 @@ describe("Artwork", () => {
 
     expect(environment.mock.getAllOperations()).toHaveLength(2)
```

Comment:
This one is also not needed with the below assertion, but if you prefer to be explicit that’s fine 👍